.github/scripts/crib/pr-comment-crib-env.js

```diff
@@ -1,8 +1,23 @@
 #!/usr/bin/env node
 
-const core = require("@actions/core");
-const github = require("@actions/github");
-const { Octokit } = require("@octokit/rest");
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+import { Octokit } from "@octokit/rest";
+import { route53RecordsExist } from "./lib/check-route53-records.js";
+
+function generateSubdomains(subdomainPrefix, prNumber) {
+  return [
+    `${subdomainPrefix}-${prNumber}-node1`,
+    `${subdomainPrefix}-${prNumber}-node2`,
+    `${subdomainPrefix}-${prNumber}-node3`,
+    `${subdomainPrefix}-${prNumber}-node4`,
+    `${subdomainPrefix}-${prNumber}-node5`,
+    `${subdomainPrefix}-${prNumber}-node6`,
```

Comment:
for loop? :D

Reply:
6733ce0fbe4e18b8c5d157598460f5af9dffe8a5